app/views/admin_users/employees/index.html.erb

```diff
@@ -12,6 +13,7 @@
     <tbody>
       <% employees.each do |employee| %>
         <tr class="text-center">
+          <td><%= employee.first_name + ' ' + employee.last_name %></td>
```

Comment:
First of all, thanks for CR! 👍 In index.html.erb file I decided to not use a full_name method because this method shows employee's email or first & last name. On this page I want to give admins both of this values. First column of the table contains first & last name wheras second column contains employee's email.